docs/reference/addons/velaux.md

```diff
@@ -2,6 +2,9 @@
 title: VelaUX
 ---
 
+import Tabs from '@theme/Tabs';
```

Comment:
If I remember it correctly, this feature is supported by mdx file. Have you preview the changes?

Reply:
Yes, I have previewed the changes and it was working fine.

Reply:
Please attach it in the PR descriptions. That's always good for reviewers to understand the effect.

Reply:
Oh sorry. I missed it. I will attach

Reply:
Attached.